demo/emails/amazon-review.tsx

```diff
@@ -0,0 +1,224 @@
+import {
+  Body,
+  Container,
+  Column,
+  Head,
+  Hr,
+  Html,
+  Img,
+  Link,
+  Preview,
+  Row,
+  Section,
+  Text,
+} from "@react-email/components";
+import * as React from "react";
+
+interface AmazonReviewEmailProps {
+  titleText?: string;
+  reviewText?: string;
+  reviwStars?: string[];
+  socialMediaIcons?: string[];
+}
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "";
+
+export const AmazonReviewEmail = ({
+  titleText = "Your opinion is important!",
+  reviewText = "Zeno Rocha, do you have a moment? We would like to know if everything went well for you. Take a moment to review your most recent purchases.",
+  reviwStars = Array(5).fill(`${baseUrl}/static/amazon-rating.gif`),
+  socialMediaIcons = [
+    `${baseUrl}/static/amazon-instagram.jpg`,
+    `${baseUrl}/static/amazon-facebook.jpg`,
+    `${baseUrl}/static/amazon-twitter.jpg`,
+  ],
+}: AmazonReviewEmailProps) => {
+  return (
+    <Html>
+      <Head />
+      <Preview>Amazon Review</Preview>
+
+      <Body style={main}>
+        <Container style={container}>
+          <Section>
+            <Column align="center">
+              <Link href="https://www.amazon.com">
+                <Img
+                  src={`${baseUrl}/static/amazon-prime-logo.png`}
+                  width="109"
+                  height="48"
+                  alt="Amazon Prime Logo"
+                />
+              </Link>
+            </Column>
+          </Section>
+
+          <Section>
+            <Column align="center">
+              <Text style={title}>{titleText}</Text>
+              <Text>{reviewText}</Text>
+            </Column>
+          </Section>
+
+          <Section>
+            <Column>
```

Comment:
The `Column` should only be used inside of a `Row` component.

This is with `Section > Column` — As you can see the markup is wrong, since there's a `td` inside of a `td`.
```tsx
<table align="center" width="100%" data-id="react-email-section" border="0" cellPadding="0" cellSpacing="0" role="presentation">
  <tbody>
    <tr>
      <td>
        <td align="center" data-id="__react-email-column">
          <p data-id="react-email-text" style="font-size:36px;line-height:38px;margin:20px 0;color:#232f3e;font-weight:400">...</p>
        </td>
      </td>
    </tr>
  </tbody>
</table>
```

This is with `Row > Column`
```tsx
<table align="center" width="100%" data-id="react-email-section" border="0" cellPadding="0" cellSpacing="0" role="presentation">
  <tbody>
    <tr>
      <td align="center" data-id="__react-email-column">
        <p data-id="react-email-text" style="font-size:36px;line-height:38px;margin:20px 0;color:#232f3e;font-weight:400">...</p>
      </td>
    </tr>
  </tbody>
</table>
```

Reply:
Ok! I will fix this